lib/avo/fields/files_field.rb

```diff
@@ -32,12 +32,9 @@ def to_permitted_param
       def fill_field(record, key, value, params)
         return record unless record.methods.include? key.to_sym
 
-        value.each do |file|
-          # Skip empty values
-          next unless file.present?
-
-          record.send(key).attach file
-        end
+        # attach files in one call to avoid index_active_storage_attachments_uniqueness violation
+        files = value.compact_blank
+        record.send(key).attach(files) unless files.none?
```

Comment:
On second thoughts, this looks better. Let me just check that it's the same.
```suggestion
        record.send(key).attach(value.compact_blank)
```